model/device.go

```diff
@@ -93,6 +93,146 @@ const (
 	DevTypeTest       = "Test"
 )
 
+// Consts containing defaults for all device configs.
+const (
+	defaultActPeriod int64 = 60
+	defaultMonPeriod int64 = 60
+)
+
+// Consts containing defaults for a controller config.
+const (
+	DefaultControllerInputs  string = "A0,X50,X51,X60,X10"
```

Comment:
It might be worth commenting what exactly these are. 

It might even be worth having a "string enum" to define these, and then have a factory to provide the default set e.g.

```
type Input string

type ExternalInput Input

const (
    SurfaceTemperature ExternalInput = "X50"
    Humidity ExternalInput = "X51"
    WaterTemperature ExternalInput = "X60"
    // etc.... 
)

type AnalogInput Input

const (
    BatteryVoltage AnalogInput = "A0"
    // etc ...
)

func defaultInputs() []Input {
    return []Input{BatteryVoltage,SurfaceTemperature,Humidity,...}
}
```

Reply:
> type Input string
> 
> type ExternalInput Input
> 
> const (
>     SurfaceTemperature ExternalInput = "X50"
>     Humidity ExternalInput = "X51"
>     WaterTemperature ExternalInput = "X60"
>     // etc.... 
> )
> 
> type AnalogInput Input
> 
> const (
>     BatteryVoltage AnalogInput = "A0"
>     // etc ...
> )

I am going to make this a separate issue to add these values to the model/sensor to then be used in this change.